src/modules/Resources/Profile.ts

```diff
@@ -303,6 +303,28 @@ class Profile extends TagoIOModule<GenericModuleParams> {
     return result;
   }
 
+  /**
+   * Renew and retrieves a new token for 6 hours
+   * @param profileID Profile ID
+   * @param token Token params for new token
+   */
+  public async renewProfileToken(profileID: GenericID, token: GenericToken): Promise<TokenCreateResponse> {
+    let result = await this.doRequest<TokenCreateResponse>({
+      path: `/profile/${profileID}/token/renew`,
+      method: "POST",
+      body: {
+        name: "Generated by TagoIO Renew",
+        profile_id: profileID,
+        origin: "admin",
+        old_token: token,
```

Comment:
Small things to consider in the request `body`:

-  `profile_id` property is already in route params
- `old_token` is already in the headers

Usage:

```ts
// current version
const resource = new Resources({ token: "my-token" });
await resource.profiles.renewProfileToken(profileID, "my-token");

// proposed changes
const resource = new Resources({ token: "my-token" });
await resource.profiles.renewProfileToken(profileID);
```

Reply:
@bgelatti if this was already discussed, just let me know!

Reply:
the code of the backend was implemented in that way, to change this, need to change de back too and probably the front

Reply:
done